validate.lic

```diff
@@ -57,6 +57,16 @@ class DRYamlValidator
     warn('You have no weapons configured in weapon_training: this will likely cause problems.')
   end
 
+  def assert_that_cyclic_no_release_spell_is_invalid_unless_listed_in_base_spells(settings)
+    return if !settings.cyclic_no_release
+
+    settings.cyclic_no_release.each do |spell_name|
+      if !get_data('spells').spell_data.include?(spell_name)
+        warn ("#{spell_name} in cyclic_no_release is not a valid spell name.")
```

Comment:
Warn or error?

Reply:
I've changed it to error. 

```
 [validate: ERROR:< garbled in cyclic_no_release is not a valid spell name.  >]                                                                                                                                                               
```

for 

```
cyclic_no_release:
  - Regenerate
  - garbled
```

In my empath's config.